### ISSUE TITLE
BLEU-4 meaning？

### ISSUE THREAD

**Issue by LzhinFdu** [tal-ai/MaKE_EMNLP2021#4]
BLEU-4 represents the 4-gram accuracy between prediction and ground truth, which is usually low on generative tasks. It is mentioned in the paper that the BLEU-4 score of the seq2seq model can reach 20+，which seems not reasonable. Is it because the BLEU score was falsely reported as BLEU-4 score？

**Comment by TjoyLiu** [comment]
> 

Hi, thanks a lot for your interest. It is BLEU-4, namely the 4-gram overlap score. I am not sure where this conclusion comes from that seq2seq model cannot reach 20+ BLEU-4 score. You can refer to these papers (https://arxiv.org/pdf/2009.11692v1.pdf, https://aclanthology.org/2022.acl-long.475.pdf, etc.). All of them have reported over 20+ BLEU-4 score on seq2seq models. Also, you can run validation on our provided dataset, if there are some other issues, I will give feedback at once.

**Comment by TjoyLiu** [comment]
> BLEU-4 represents the 4-gram accuracy between prediction and ground truth, which is usually low on generative tasks. It is mentioned in the paper that the BLEU-4 score of the seq2seq model can reach 20+，which seems not reasonable. Is it because the BLEU score was falsely reported as BLEU-4 score？
Hi, thanks a lot for your interest. It is BLEU-4, namely the 4-gram overlap score. I am not sure where this conclusion comes from that seq2seq model cannot reach 20+ BLEU-4 score. You can refer to these papers (https://arxiv.org/pdf/2009.11692v1.pdf, https://aclanthology.org/2022.acl-long.475.pdf, etc.). All of them have reported over 20+ BLEU-4 score on seq2seq models. Also, you can run validation on our provided dataset, if there are some other issues, I will give feedback at once.

**Comment by TjoyLiu** [comment]
> BLEU-4 represents the 4-gram accuracy between prediction and ground truth, which is usually low on generative tasks. It is mentioned in the paper that the BLEU-4 score of the seq2seq model can reach 20+，which seems not reasonable. Is it because the BLEU score was falsely reported as BLEU-4 score？

Hi, thanks a lot for your interest. It is BLEU-4, namely the 4-gram overlap score. I am not sure where this conclusion comes from that seq2seq model cannot reach 20+ BLEU-4 score. You can refer to these papers (https://arxiv.org/pdf/2009.11692v1.pdf, https://aclanthology.org/2022.acl-long.475.pdf, etc.). All of them have reported over 20+ BLEU-4 score on seq2seq models. Also, you can run validation on our provided dataset, if there are some other issues, I will give feedback at once.

**Comment by LzhinFdu** [comment]
In fact when I use the fine-tuned mT5 model on this dataset, although the generated problems seem smooth and logical, the BLEU score is only 4.1. So I'm curious why the baseline model transformer can achieve better BLEU-4 than mT5. Maybe something went wrong during the model training phase. I'll check it out later. 
Anyway thanks for your reply.

**Comment by LzhinFdu** [comment]
Thanks for your reply. Does the 4-gram in this paper refer to four Chinese characters? I used word-level BLEU-4 in my evaluation, which may be the reason why I got a very low BLEU-4 score.

**Comment by TjoyLiu** [comment]
Hi

I have also tried some experiments with mT5, this model performs badly on
this task. This phenomenon is also seen for other multilingual pre-training
models.
